docs/src/paradox/03-apis/api-v2/tei-xml.md

```diff
@@ -0,0 +1,433 @@
+<!---
+Copyright © 2015-2018 the contributors (see Contributors.md).
+
+This file is part of Knora.
+
+Knora is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Knora is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+# TEI/XML: Converting Standoff to TEI/XML
+
+@@toc
+
+## General
+
+Knora offers a way to convert standoff markup to TEI/XML. The conversion is based on the assumption that a whole resource is to be turned into a TEI document.
+There is a basic distinction between the body and the header of a TEI document. The resource's property that contains the text with standoff markup is mapped to the TEI document's body.
+Other of the resource's property may be mapped to the TEI header.
+
+## Standard Standoff to TEI Conversion
+
+Knora offers a built-in conversion form standard standoff entities (defined in `standoff-onto`) tags to TEI.
```

Comment:
`standoff-onto` -> the `standoff` ontology

(its name is just `standoff`)

Reply:
done in https://github.com/dhlab-basel/Knora/pull/862/commits/670ab38f9277c2bc7874457c93c5e032357bb114